db/schema.rb

```diff
@@ -10,11 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_08_10_100000) do
+ActiveRecord::Schema.define(version: 2023_09_27_073435) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "hstore"
   enable_extension "pg_trgm"
+  enable_extension "pgcrypto"
```

Comment:
I am not sure about the `pgcrypto` extension. 

To my understanding it should be in the schema.rb since it has been introduced in `20220712100010_add_segment_klass_identifier.rb` 
Besides is inserted every time I run `db:migrate` so I decided to leave it.